decidim-core/spec/forms/account_form_spec.rb

```diff
@@ -155,6 +156,19 @@ module Decidim
       end
     end
 
+    describe "password_confirmation" do
```

Comment:
Can you add this same validation in the registration spec too? It's on decidim-core/spec/forms/registration_form_spec.rb

Reply:
done